dockstore-common/src/main/java/io/dockstore/common/yaml/Service12.java

```diff
@@ -30,6 +30,8 @@ public class Service12 extends AbstractYamlService implements Workflowish {
 
     private DescriptorLanguageSubclass subclass;
 
+    private boolean enableAutomaticDoiCreation = true;
```

Comment:
nitpick, since this is the default, maybe this should be flipped to `disableAutomaticDoiCreation` or `enableEndUserDoiCreation`